pkts-core/src/main/java/io/pkts/framer/RTPFramer.java

```diff
@@ -98,9 +98,7 @@ public RtpPacket frame(final TransportPacket parent, final Buffer buffer) throws
             final int csrcCount = b & 0x0F;
 
             if (hasExtension) {
-                final short extensionHeaders = buffer.readShort();
                 final int length = buffer.readUnsignedShort();
-                final Buffer extensionData = buffer.readBytes(length);
```

Comment:
Same goes for this one. We need to read the bytes off of the buffer or it won't work... did the unit tests actually pass for these changes?